spec/requests/user/user_show_spec.rb

```diff
@@ -1,52 +1,71 @@
 require "rails_helper"
 
 RSpec.describe "UserShow", type: :request do
-  let_it_be(:user) { create(:user, email_public: true, currently_hacking_on: "JSON-LD", education: "DEV University") }
+  let_it_be(:user) { create(:user, email_public: true, education: "DEV University") }
+  let(:doc) { Nokogiri::HTML(response.body) }
+  let(:text) { doc.at('script[type="application/ld+json"]').text }
+  let(:response_json) { JSON.parse(text) }
+  let(:main_entity_of_page) { { "@type" => "WebPage", "@id" => URL.user(user) } }
+  let(:same_as) do
+    [
+      "https://twitter.com/#{user.twitter_username}",
+      "https://github.com/#{user.github_username}",
+      user.mastodon_url,
+      user.facebook_url,
+      user.youtube_url,
+      user.linkedin_url,
+      user.behance_url,
+      user.stackoverflow_url,
+      user.dribbble_url,
+      user.medium_url,
+      user.gitlab_url,
+      user.instagram_url,
+      user.twitch_username,
+      user.website_url,
+    ]
+  end
+
+  let(:disambiguating_description) do
+    [
+      user.mostly_work_with,
+      user.currently_hacking_on,
+      user.currently_learning,
+    ]
+  end
+
+  let(:works_for) { [{ "@type" => "Organization", "name" => user.employer_name, "url" => user.employer_url }] }
 
   describe "GET /:slug (user)" do
     before do
-      user.update_columns(employment_title: "SEO", employer_name: "DEV", linkedin_url: "www.linkedin.com/company/example/")
+      user.update_columns(employment_title: "SEO", employer_name: "DEV", employer_url: "www.dev.to")
+      user.update_columns(currently_learning: "Preact", mostly_work_with: "Ruby", currently_hacking_on: "JSON-LD")
+      user.update_columns(mastodon_url: "www.example.com", facebook_url: "www.facebook.com/example", linkedin_url: "www.linkedin.com/company/example/")
+      user.update_columns(youtube_url: "www.youtube.com/example", behance_url: "www.behance.com/example", stackoverflow_url: "www.stackoverflow.com/example")
+      user.update_columns(dribbble_url: "www.dribbble.com/example", medium_url: "www.medium.com/example", gitlab_url: "www.gitlab.com/example")
+      user.update_columns(instagram_url: "www.instagram.com/example", twitch_username: "Example007", website_url: "www.example.com/example")
```

Comment:
Could probably refactor this into calling it just once, and passing the arguments all at once? Like:
```
user.update_columns(
  employment_title: "SEO",
  employer_name: "DEV",
  # etc etc
)
```

Reply:
Excellent suggestion! I just made this change and the spec looks so much nicer. 😂

Reply:
You could also expand user factory with a new transient.  `create(:user, :with_all_info)`  would be pretty clean

Reply:
I like that idea as well--trying it out now! 😄